build/charts/antrea/templates/controller/clusterrole.yaml

```diff
@@ -206,6 +206,17 @@ rules:
       - patch
       - create
       - delete
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - packetsamplings
```

Comment:
I didn't find any code updating its Spec, so neither component needs patch/update permission of the resource itself, you should separate them to ensure least privillege.

Reply:
Sure。Updated